src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/validation/FlowValidator.java

```diff
@@ -203,9 +203,10 @@ public void validateForSwapEndpoints(RequestedFlow firstFlow, RequestedFlow seco
 
     @VisibleForTesting
     void checkBandwidth(RequestedFlow flow) throws InvalidFlowException {
-        if (flow.getBandwidth() < 0) {
+        if (flow.getBandwidth() != 0 && flow.getBandwidth() < 64) {
```

Comment:
Is it valid for all switch models?

Reply:
It is valid for all NoviFlow switch models

Reply:
Now this limitation is taken into account only for NoviFlow switches.